admin/src/authProvider.js

```diff
@@ -0,0 +1,40 @@
+import api from "@services/api";
+
+export default {
+  login: async (params) => {
+    try {
+      const formData = {
+        email: params.username,
+        password: params.password,
+      };
+      const { data } = await api.post(`/auth/login`, formData);
+      localStorage.setItem("token", data.token);
```

Comment:
NB: You know how to register a token in a LocalStorage (as an answer to this morning's stand-up)